src/run-view-closed.ts

```diff
@@ -27,16 +19,13 @@ export const RunViewClosed = async (
     );
   }
 
+  const viewClosedBody = baseHandlerArgs.body as ViewClosedInvocationBody;
   // We don't catch any errors the handlers may throw, we let them throw, and stop the process
   // deno-lint-ignore no-explicit-any
   const response: any = await handler({
-    inputs,
-    env,
-    token,
-    team_id,
-    enterprise_id,
-    body,
-    view,
+    ...baseHandlerArgs,
+    body: viewClosedBody,
+    view: viewClosedBody.view,
```

Comment:
I do like the change in this PR, like in this section, where now it is obvious what event-specific properties are being tacked on to userland handler arguments before invoking user code.